### ISSUE TITLE
 [RTL] The checkmark icon on the Tracking Protection in the Settings menu are is located on the wrong side

### ISSUE THREAD

**Issue by SimonBasca** [mozilla-mobile/firefox-ios#6541]
See screenshot:
https://drive.google.com/file/d/16BvqydfmxI1a5BSye417OsVLCG8Bbtvh/view

The checkmark icon should be located on the right side.

Via : https://bugzilla.mozilla.org/show_bug.cgi?id=1634931

┆Issue is synchronized with this [Jira Task](https://mozilla-hub.atlassian.net/browse/FXIOS-2396)

**Comment by data-sync-user** [comment]
➤ Simion Basca commented:

Verifying as fix on main 228b77b9a89f133417cb748fcf720fb01e6b6f1a

**Comment by SimonBasca** [comment]
Verified fix on 98.0 (8023)
